WordPress/src/main/res/values/styles.xml

```diff
@@ -559,4 +559,13 @@
         <item name="android:textSize">@dimen/text_sz_medium</item>
         <item name="android:textStyle">bold</item>
     </style>
+
+    <style name="UploadProgressBar" parent="Widget.AppCompat.ProgressBar.Horizontal">
+        <item name="android:indeterminateOnly">false</item>
+        <item name="android:progressDrawable">@drawable/wp_progress_horizontal</item>
```

Comment:
I haven't heard of the missing progress bar problem, but any chance [this](https://www.bountysource.com/issues/35649135-circular-indeterminate-progress-bar-not-working-on-android-4-x) explains it?